x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/queries/TokenPruningConfig.java

```diff
@@ -47,8 +47,8 @@ public TokenPruningConfig(float tokensFreqRatioThreshold, float tokensWeightThre
             throw new IllegalArgumentException(
                 "["
                     + TOKENS_FREQ_RATIO_THRESHOLD.getPreferredName()
-                    + "] must be between [1.0] and ["
-                    + String.format(Locale.ROOT, "%.1f", MAX_TOKENS_FREQ_RATIO_THRESHOLD)
+                    + "] must be between [1] and ["
+                    + String.format(Locale.ROOT, "%d", (int) MAX_TOKENS_FREQ_RATIO_THRESHOLD)
```

Comment:
It turns out that this value is read as an `int` (see line 157). This was a good catch in the clients PR. I kept the float value for the class here because it's already been committed and it didn't seem worth bumping the transport version to fix it, but I cleaned up external communication that refers to it as a float in docs and clients.